bin/generate

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env sh
+
+set -e
+
+example_name="$@"
```

Comment:
@MichielBijl the convention I've seen in shell scripts is to use underscores. Google's styleguide suggests using underscorized variable names (as with function names) https://google.github.io/styleguide/shell.xml?showone=Function_Names#Function_Names